api/openapi/model-registry.yaml

```
@@ -1023,10 +1023,18 @@ components:
         - $ref: "#/components/schemas/BaseResourceList"
     RegisteredModelCreate:
       description: A registered model in model registry. A registered model has ModelVersion children.
+      required:
+        - name
       allOf:
         - type: object
         - $ref: "#/components/schemas/BaseResourceCreate"
         - $ref: "#/components/schemas/RegisteredModelUpdate"
+          properties:
+            name:
+              description: |-
+                The client provided name of the context. It must be unique among all the contexts of the same
+                type within a database instance and cannot be changed once set.
```

This description is a bit a leaky abstraction.
Let's change it to use the domain model for Model Registry.

```suggestion
                The client provided name of the model. It must be unique among all the RegisteredModels of the same
                type within a Model Registry instance and cannot be changed once set.
```